test/integration/order_integration_test.rb

```diff
@@ -27,21 +28,13 @@ def setup
   end
 
   test "user can view page for specific order" do
-    skip
     ApplicationController.any_instance.stubs(:current_user).returns(user)
     click_link_or_button 'Cart'
     click_link_or_button 'Order History'
     click_link_or_button "Order #{order.id}"
     assert current_path, user_order_path(user, order)
-    within ('#order_history') do
+    within ('#history') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.